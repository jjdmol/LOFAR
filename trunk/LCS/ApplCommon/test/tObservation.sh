#!/bin/sh
./runctest.sh tObservation
