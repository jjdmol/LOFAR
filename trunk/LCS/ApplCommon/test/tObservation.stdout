>>>

Observation  : 
ObsID        : 5025
starttime    : 2008-Dec-16 15:30:00
stoptime     : 2008-Dec-16 15:34:00
stations     : [CS016,CS017,CS018]
antennaArray : HBA
antenna set  : HBA_DUAL
receiver set : 000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000111111111111
sampleClock  : 160
filter       : HB_210_240
splitter     : ON
nyquistZone  : 3

(Receivers)  : [0..11]
Stations     : [CS016,CS017,CS018]
BLG nodes    : [bgl001..bgl003]
Storage nodes: [list001,list002]

nrBeams      : 1
Beam[0].name       : observation[5025]beam[0]
Beam[0].target     : 3C381
Beam[0].antennaSet : HBA_ZERO
Beam[0].momID      : 0
Beam[0].subbandList: [2,3,4,5]
Beam[0].beamletList: [0,1,2,3]
nrPointings : 1
Beam[0].pointing[0]: 6.000000, 0.500000, J2000, 2008-Dec-16 15:30:00
nrTABs      : 2
nrTABrings  : 3
TABringsize : 0.023
Beam[0].TAB[0]: 6.010000, 0.510000, J2000, 0.000000, coherent
Beam[0].TAB[1]: 5.990000, 0.490000, J2000, 20.000000, incoherent
beamlet2beams   : [0,0,0,0,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,999,999,999,999,999,999,999,999,999,999,999,999,999,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,999,999,999,999,999,999,999,999,999,999,999,999,999,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,999,999,999,999,999,999,999,999,999,999,999,999,999,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,999,999,999,999,999,999,999,999,999,999,999,999,999]
nrAnaBeams   : 0


Observation  : 
ObsID        : 5025
starttime    : 2008-Dec-16 15:30:00
stoptime     : 2008-Dec-16 15:34:00
stations     : [CS016,CS017,CS018]
antennaArray : HBA
antenna set  : HBA_JOINED
receiver set : 000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000111111111111
sampleClock  : 160
filter       : HB_210_240
splitter     : OFF
nyquistZone  : 3

(Receivers)  : [0..11]
Stations     : [CS016,CS017,CS018]
BLG nodes    : [bgl001..bgl003]
Storage nodes: [list001,list002]

nrBeams      : 1
Beam[0].name       : observation[5025]beam[0]
Beam[0].target     : 3C381
Beam[0].antennaSet : HBA_JOINED
Beam[0].momID      : 0
Beam[0].subbandList: [2,3,4,5]
Beam[0].beamletList: [0,1,2,3]
nrPointings : 1
Beam[0].pointing[0]: 6.000000, 0.500000, J2000, 2008-Dec-16 15:30:00
nrTABs      : 2
nrTABrings  : 3
TABringsize : 0.023
Beam[0].TAB[0]: 6.010000, 0.510000, J2000, 0.000000, coherent
Beam[0].TAB[1]: 5.990000, 0.490000, J2000, 20.000000, incoherent
beamlet2beams   : [0,0,0,0,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,999,999,999,999,999,999,999,999,999,999,999,999,999,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,999,999,999,999,999,999,999,999,999,999,999,999,999,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,999,999,999,999,999,999,999,999,999,999,999,999,999,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,999,999,999,999,999,999,999,999,999,999,999,999,999]
nrAnaBeams   : 0

<<<

Observation  : 
ObsID        : 5025
starttime    : 2008-Dec-16 15:30:00
stoptime     : 2008-Dec-16 15:34:00
stations     : [CS016,CS017,CS018]
antennaArray : HBA
antenna set  : HBA_JOINED
receiver set : 000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000111111111111
sampleClock  : 160
filter       : HB_210_240
splitter     : OFF
nyquistZone  : 3

(Receivers)  : [0..11]
Stations     : [CS016,CS017,CS018]
BLG nodes    : [bgl001..bgl003]
Storage nodes: [list001,list002]

nrBeams      : 2
Beam[0].name       : observation[5025]beam[0]
Beam[0].target     : 3C381
Beam[0].antennaSet : HBA_JOINED
Beam[0].momID      : 0
Beam[0].subbandList: [2,3,4,5]
Beam[0].beamletList: [0,1,2,3]
nrPointings : 1
Beam[0].pointing[0]: 6.000000, 0.500000, J2000, 2008-Dec-16 15:30:00
nrTABs      : 2
nrTABrings  : 3
TABringsize : 0.023
Beam[0].TAB[0]: 6.010000, 0.510000, J2000, 0.000000, coherent
Beam[0].TAB[1]: 5.990000, 0.490000, J2000, 20.000000, incoherent
Beam[1].name       : observation[5025]beam[1]
Beam[1].target     : 
Beam[1].antennaSet : HBA_JOINED
Beam[1].momID      : 0
Beam[1].subbandList: [4,3,102]
Beam[1].beamletList: [15,16,18]
nrPointings : 1
Beam[1].pointing[0]: 0.234568, 0.123457, AZEL, 2008-Dec-16 15:30:00
nrTABs      : 0
nrTABrings  : 0
TABringsize : 0
beamlet2beams   : [0,0,0,0,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,1,1,-1,1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,999,999,999,999,999,999,999,999,999,999,999,999,999,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,999,999,999,999,999,999,999,999,999,999,999,999,999,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,999,999,999,999,999,999,999,999,999,999,999,999,999,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,999,999,999,999,999,999,999,999,999,999,999,999,999]
nrAnaBeams   : 0

>>>
INFO  LCS.ApplCommon - Clock of observation 5025 and 5026 conflict
INFO  LCS.ApplCommon - Conflicting use of receivers between observation 5025 and 5027
DEBUG LCS.ApplCommon - receiverConflict: 000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000111100000000
INFO  LCS.ApplCommon - Conflict in beamlets between observation 5025 and 5028
DEBUG LCS.ApplCommon - First conflicting beamlet: 2
INFO  LCS.ApplCommon - Conflict in nrSlotsInFrame: 48<->54 for resp. observation 5025 and 5029
<<<
No conflict found in file 5 which is oke.
getRCUbitset(96,48,'') = 000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000111111111111
getRCUbitset(96,96,'') = 000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000111111111111
getRCUbitset(96,48,LBA_XXX) = 000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000111111111111
getRCUbitset(96,96,LBA_XXX) = 000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000111111111111
getRCUbitset(96,48,HBA_XXX) = 000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000111111111111
getRCUbitset(96,96,HBA_XXX) = 000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000111111111111
HBA_ZERO(false) = HBA
HBA_ZERO(true)  = HBA0
HBA_ONE(false) = HBA
HBA_ONE(true)  = HBA1
HBA_DUAL(false) = HBA
HBA_DUAL(true)  = HBA0
HBA_JOINED(false) = HBA
HBA_JOINED(true)  = HBA
LBA_INNER(false) = LBA
LBA_INNER(true)  = LBA
LBA_OUTER(false) = LBA
LBA_OUTER(true)  = LBA
LBA_X(false) = LBA
LBA_X(true)  = LBA
>>>
OLD SYNTAX

Observation  : 
ObsID        : 5025
starttime    : 2008-Dec-16 15:30:00
stoptime     : 2008-Dec-16 15:34:00
stations     : [CS001,CS004,RS005]
antennaArray : HBA
antenna set  : HBA_DUAL
receiver set : 000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000111111111111
sampleClock  : 160
filter       : HBA_210_250
splitter     : ON
nyquistZone  : 3

(Receivers)  : [0..11]
Stations     : [CS001,RS005,CS004]
BLG nodes    : [bgl001..bgl003]
Storage nodes: [list001,list002]

nrBeams      : 4
Beam[0].name       : observation[5025]beam[0]_0
Beam[0].target     : 
Beam[0].antennaSet : HBA_ZERO
Beam[0].momID      : 0
Beam[0].subbandList: [2,3,4,5]
Beam[0].beamletList: [0,1,2,3]
nrPointings : 1
Beam[0].pointing[0]: 6.000000, 0.500000, J2000, 2008-Dec-16 15:30:00
nrTABs      : 0
nrTABrings  : 0
TABringsize : 0
Beam[1].name       : observation[5025]beam[0]_1
Beam[1].target     : 
Beam[1].antennaSet : HBA_ONE
Beam[1].momID      : 0
Beam[1].subbandList: [2,3,4,5]
Beam[1].beamletList: [0,1,2,3]
nrPointings : 1
Beam[1].pointing[0]: 6.000000, 0.500000, J2000, 2008-Dec-16 15:30:00
nrTABs      : 0
nrTABrings  : 0
TABringsize : 0
Beam[2].name       : observation[5025]beam[1]_0
Beam[2].target     : 
Beam[2].antennaSet : HBA_ZERO
Beam[2].momID      : 0
Beam[2].subbandList: [20,21,22,23,24,25,40,41,42,43,44]
Beam[2].beamletList: [45,46,47,61,62,63,64,65,66,67,68]
nrPointings : 1
Beam[2].pointing[0]: 1.000000, 1.500000, J2000, 2008-Dec-16 15:30:00
nrTABs      : 0
nrTABrings  : 0
TABringsize : 0
Beam[3].name       : observation[5025]beam[1]_1
Beam[3].target     : 
Beam[3].antennaSet : HBA_ONE
Beam[3].momID      : 0
Beam[3].subbandList: [20,21,22,23,24,25,40,41,42,43,44]
Beam[3].beamletList: [45,46,47,61,62,63,64,65,66,67,68]
nrPointings : 1
Beam[3].pointing[0]: 1.000000, 1.500000, J2000, 2008-Dec-16 15:30:00
nrTABs      : 0
nrTABrings  : 0
TABringsize : 0
beamlet2beams   : [0,0,0,0,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,1,1,1,999,999,999,999,999,999,999,999,999,999,999,999,999,1,1,1,1,1,1,1,1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,999,999,999,999,999,999,999,999,999,999,999,999,999,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,999,999,999,999,999,999,999,999,999,999,999,999,999,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,999,999,999,999,999,999,999,999,999,999,999,999,999]
nrAnaBeams   : 0

NEW SYNTAX

Observation  : 
ObsID        : 5025
starttime    : 2008-Dec-16 15:30:00
stoptime     : 2008-Dec-16 15:34:00
stations     : [CS001,CS004,RS005]
antennaArray : HBA
antenna set  : HBA_DUAL
receiver set : 000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000111111111111
sampleClock  : 160
filter       : HBA_210_250
splitter     : ON
nyquistZone  : 3

(Receivers)  : [0..11]
Stations     : [CS001,RS005,CS004]
BLG nodes    : [bgl001..bgl003]
Storage nodes: [list001,list002]

nrBeams      : 4
Beam[0].name       : observation[5025]beam[0]_0
Beam[0].target     : 
Beam[0].antennaSet : HBA_ZERO
Beam[0].momID      : 0
Beam[0].subbandList: [2,3,4,5]
Beam[0].beamletList: [0,1,2,3]
nrPointings : 1
Beam[0].pointing[0]: 6.000000, 0.500000, J2000, 2008-Dec-16 15:30:00
nrTABs      : 0
nrTABrings  : 0
TABringsize : 0
Beam[1].name       : observation[5025]beam[0]_1
Beam[1].target     : 
Beam[1].antennaSet : HBA_ONE
Beam[1].momID      : 0
Beam[1].subbandList: [2,3,4,5]
Beam[1].beamletList: [0,1,2,3]
nrPointings : 1
Beam[1].pointing[0]: 6.000000, 0.500000, J2000, 2008-Dec-16 15:30:00
nrTABs      : 0
nrTABrings  : 0
TABringsize : 0
Beam[2].name       : observation[5025]beam[1]_0
Beam[2].target     : 
Beam[2].antennaSet : HBA_ZERO
Beam[2].momID      : 0
Beam[2].subbandList: [20,21,22,23,24,25,40,41,42,43,44]
Beam[2].beamletList: [45,46,47,61,62,63,64,65,66,67,68]
nrPointings : 1
Beam[2].pointing[0]: 1.000000, 1.500000, J2000, 2008-Dec-16 15:30:00
nrTABs      : 0
nrTABrings  : 0
TABringsize : 0
Beam[3].name       : observation[5025]beam[1]_1
Beam[3].target     : 
Beam[3].antennaSet : HBA_ONE
Beam[3].momID      : 0
Beam[3].subbandList: [20,21,22,23,24,25,40,41,42,43,44]
Beam[3].beamletList: [45,46,47,61,62,63,64,65,66,67,68]
nrPointings : 1
Beam[3].pointing[0]: 1.000000, 1.500000, J2000, 2008-Dec-16 15:30:00
nrTABs      : 0
nrTABrings  : 0
TABringsize : 0
beamlet2beams   : [0,0,0,0,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,1,1,1,999,999,999,999,999,999,999,999,999,999,999,999,999,1,1,1,1,1,1,1,1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,999,999,999,999,999,999,999,999,999,999,999,999,999,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,999,999,999,999,999,999,999,999,999,999,999,999,999,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,999,999,999,999,999,999,999,999,999,999,999,999,999]
nrAnaBeams   : 0

<<<
